This PR has improvements in navbar

harrydigos opened the pull request.

panteliselef commented on components/navbar/NavbarProvider.tsx:
```diff
@@ -0,0 +1,26 @@
+    const currentYOffset = window.pageYOffset;
+
+    setYOffset(currentYOffset);
+    setVisible(yOffset > currentYOffset || currentYOffset < 64);
```
🚫`setVisible((oldValue=> {...}))`

panteliselef commented on components/navbar/NavbarProvider.tsx:
```diff
@@ -0,0 +1,26 @@
+  useEffect(() => {
+    window.addEventListener("scroll", handleScroll);
+    return () => window.removeEventListener("scroll", handleScroll);
+  }, [yOffset]);
```
yOffset should be removed

harrydigos committed:
Fixes + hiding navbar on scroll


User provider & cleaned navbar code


Fixed on scroll


User Cart in Navbar


User cart in local storage & buy product


User cart in local storage & buy product

panteliselef committed:
Use a callback to store cart in localStorage
+ Removing unnecessary state for the Navber animation
